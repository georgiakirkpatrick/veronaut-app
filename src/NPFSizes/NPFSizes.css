.NPFSizes .FormCheckboxSection__options-container {
    height: 55vh;
}

@media screen and (min-width: 360px) {
    .NPFSizes .FormCheckboxSection__options-container {
        height: 50vh;
    }
}

@media screen and (min-width: 375px) {
    .NPFSizes .FormCheckboxSection__options-container {
        height: 50vh;
    }
}

@media screen and (min-width: 375px) and (min-height: 812px) {
    .NPFSizes .FormCheckboxSection__options-container {
        height: 43vh;
    }
}

@media screen and (min-width: 411px) {
    .NPFSizes .FormCheckboxSection__options-container {
        height: 45vh;
    }
}

@media screen and (min-width: 768px) {
    .NPFSizes .FormCheckboxSection {
        padding: 1vh 3vw;
        margin: 1vh 0;
    }

    .NPFSizes .FormCheckboxSection__options-container {
        height: 20vh;
    }

    .NPFSizes .FormCheckboxSection__option {
        padding: 5px;
    }
}

@media screen and (min-width: 1024px) {
    .NPFSizes .FormCheckboxSection__options-container {
        height: 30vh;
    }
}

@media screen and (min-width: 1024px) and (min-height: 1366px) {
    .NPFSizes .FormCheckboxSection {
        padding: 1vh 3vw;
    }

    .NPFSizes .FormCheckboxSection__options-container {
        height: 17vh;
    }
}

@media screen and (min-width: 1440px) {
    .NPFSizes .FormCheckboxSection {
        height: 27vh;
        padding: 1vh 3vw;
    }

    .NPFSizes .FormCheckboxSection__options-container {
        height: 22vh;
    }
}

@media screen and (min-width: 2560px) {
    .NPFSizes .FormCheckboxSection {
        height: 20vh;
        padding: 1vh 3vw;
    }

    .NPFSizes .FormCheckboxSection__options-container {
        height: 15vh;
    }
}
