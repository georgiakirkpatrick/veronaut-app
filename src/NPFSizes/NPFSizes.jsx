import React from 'react'
import FormPage from '../FormPage/FormPage'
import FormCheckboxSection from '../FormCheckboxSection/FormCheckboxSection'
import FormPromptWithSub from '../FormPromptWithSub/FormPromptWithSub'
import NPFFooter from '../NPFFooter/NPFFooter'
import './NPFSizes.css'
import sizeData from '../SIZES'

const NPFSizes = props => {
  const {
    currentPage,
    selectedSizeOptions,
    setPage,
    setSelectedSizeOptions,
    sizeSystem
  } = props

  const maternityOptions = [
    ...sizeData.women.alpha.maternity.map(size => (
      {
        id: size.id,
        text: size.text
      }
    )),
    ...Object.values(sizeData.women.dress.maternity).map((size, index) => (
      {
        id: size[sizeSystem].id,
        text: size[sizeSystem].text
      }
    ))
  ]

  const nextButton = () => {
    setPage(currentPage + 1)
  }

  const petiteOptions = [
    ...sizeData.women.alpha.petite.map(size => (
      {
          id: size.id,
          text: size.text
      }
    )),
    ...Object.values(sizeData.women.dress.petite).map((size, index) => (
      {
          id: size[sizeSystem].id,
          text: size[sizeSystem].text
      }
    ))
  ]

  const sizeChange = event => {
    setSelectedSizeOptions(
      {
        ...selectedSizeOptions,
        [event.target.id]: !selectedSizeOptions[event.target.id]
      }
    )
  }

  const standardOptions = [
    ...sizeData.women.alpha.standard.map(size => (
      {
        id: size.id,
        text: size.text
      }
    )),
    ...Object.values(sizeData.women.dress.standard).map((size, index) => (
      {
        id: size[sizeSystem].id,
        text: size[sizeSystem].text
      }
    ))
  ]

  const systemName = sizeData.systems.filter(system => system.id === sizeSystem)[0].text

  const tallOptions = [
    ...sizeData.women.alpha.tall.map(size => (
      {
        id: size.id,
        text: size.text
      }
    )),
    ...Object.values(sizeData.women.dress.tall).map((size, index) => (
      {
        id: size[sizeSystem].id,
        text: size[sizeSystem].text
      }
    ))
  ]

  return (
    <div className='NPFSizes'>
      <FormPage title='Sizes'>
          <FormPromptWithSub 
              prompt='Select all size options available on this product page'
              promptSubtitle='Include sold out sizes'
          />

          <FormCheckboxSection 
              prompt={`Standard ${systemName} sizing`}
              options={standardOptions}
              selectedOptions={selectedSizeOptions}
              handleChange={event => sizeChange(event)}
          />

          <FormCheckboxSection 
              prompt={`Petite ${systemName} sizing`} 
              options={petiteOptions}
              selectedOptions={selectedSizeOptions}
              handleChange={event => sizeChange(event)}
          />

          <FormCheckboxSection 
              prompt={`Tall ${systemName} sizing`} 
              options={tallOptions}
              selectedOptions={selectedSizeOptions}
              handleChange={event => sizeChange(event)}
          />

          <FormCheckboxSection 
              prompt={`Maternity ${systemName} sizing`} 
              options={maternityOptions}
              selectedOptions={selectedSizeOptions}
              handleChange={event => sizeChange(event)}
          />
      </FormPage>

      <NPFFooter 
        buttons='prevNext' 
        previousButton={() => setPage(currentPage - 1)}
        nextButton={() => nextButton()} 
      />
    </div>
  )
}

NPFSizes.defaultProps = {
  currentPage: 0,
  selectedSizeOptions: [],
  setPage: () => {},
  setSelectedSizeOptions: () => {},
  sizeSystem: 1
}    

export default NPFSizes