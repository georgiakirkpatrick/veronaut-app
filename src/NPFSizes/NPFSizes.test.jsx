import React from 'react'
import { render, screen } from '@testing-library/react'
import NPFSizes from './NPFSizes'
import { describe, it, expect } from 'vitest'

describe ('NPFSizes', () => {
  it ('renders without crashing', () => {
    render(<NPFSizes />)
    screen.debug()
  })

  it('renders the UI as expected', () => {
    const tree = render(<NPFSizes />)
    expect(tree).toMatchSnapshot()    
  })
})