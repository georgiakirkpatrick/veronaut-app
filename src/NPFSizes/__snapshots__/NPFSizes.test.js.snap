// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NPFSizes renders the UI as expected 1`] = `
<div
  className="NPFSizes"
>
  <section
    className="FormPage"
  >
    <h1>
      Sizes
    </h1>
    <form
      id="form-page"
    >
      <div
        className="FormPromptWithSub"
      >
        <p
          className="FormPromptWithSub__prompt"
        >
          Select all size options available on this product page
        </p>
        <p
          className="FormPromptWithSub__subtitle"
        >
          Include sold out sizes
        </p>
      </div>
      <div
        className="FormCheckboxSection"
      >
        <p>
          Standard US sizing
        </p>
        <div
          className="FormCheckboxSection__options-container"
        >
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={1}
              name="sXXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={1}
            >
              XXS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={2}
              name="sXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={2}
            >
              XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={3}
              name="sS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={3}
            >
              S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={4}
              name="sM"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={4}
            >
              M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={5}
              name="sL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={5}
            >
              L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={6}
              name="sXL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={6}
            >
              XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={7}
              name="s2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={7}
            >
              2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={8}
              name="s3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={8}
            >
              3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={9}
              name="s4X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={9}
            >
              4X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={26}
              name="s00"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={26}
            >
              00
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={27}
              name="s0"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={27}
            >
              0
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={28}
              name="s2"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={28}
            >
              2
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={29}
              name="s4"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={29}
            >
              4
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={30}
              name="s6"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={30}
            >
              6
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={31}
              name="s8"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={31}
            >
              8
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={32}
              name="s10"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={32}
            >
              10
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={33}
              name="s12"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={33}
            >
              12
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={34}
              name="s14"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={34}
            >
              14
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={35}
              name="s16"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={35}
            >
              16
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={36}
              name="s18"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={36}
            >
              18
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={37}
              name="s20"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={37}
            >
              20
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={38}
              name="s22"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={38}
            >
              22
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={39}
              name="s24"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={39}
            >
              24
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={40}
              name="s26"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={40}
            >
              26
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={41}
              name="s28"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={41}
            >
              28
            </label>
          </div>
        </div>
      </div>
      <div
        className="FormCheckboxSection"
      >
        <p>
          Petite US sizing
        </p>
        <div
          className="FormCheckboxSection__options-container"
        >
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={106}
              name="pXXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={106}
            >
              XXS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={107}
              name="pXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={107}
            >
              XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={108}
              name="pS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={108}
            >
              S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={109}
              name="pM"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={109}
            >
              M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={110}
              name="pL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={110}
            >
              L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={111}
              name="pXL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={111}
            >
              XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={112}
              name="p2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={112}
            >
              2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={113}
              name="p3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={113}
            >
              3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={114}
              name="p4X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={114}
            >
              4X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={131}
              name="p00"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={131}
            >
              00
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={132}
              name="p0"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={132}
            >
              0
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={133}
              name="p2"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={133}
            >
              2
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={134}
              name="p4"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={134}
            >
              4
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={135}
              name="p6"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={135}
            >
              6
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={136}
              name="p8"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={136}
            >
              8
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={137}
              name="p10"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={137}
            >
              10
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={138}
              name="p12"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={138}
            >
              12
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={139}
              name="p14"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={139}
            >
              14
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={140}
              name="p16"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={140}
            >
              16
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={141}
              name="p18"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={141}
            >
              18
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={142}
              name="p20"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={142}
            >
              20
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={143}
              name="p22"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={143}
            >
              22
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={144}
              name="p24"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={144}
            >
              24
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={145}
              name="p26"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={145}
            >
              26
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={146}
              name="p28"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={146}
            >
              28
            </label>
          </div>
        </div>
      </div>
      <div
        className="FormCheckboxSection"
      >
        <p>
          Tall US sizing
        </p>
        <div
          className="FormCheckboxSection__options-container"
        >
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={211}
              name="tXXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={211}
            >
              XXS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={212}
              name="tXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={212}
            >
              XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={213}
              name="tS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={213}
            >
              S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={214}
              name="tM"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={214}
            >
              M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={215}
              name="tL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={215}
            >
              L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={216}
              name="tXL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={216}
            >
              XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={217}
              name="t2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={217}
            >
              2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={218}
              name="t3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={218}
            >
              3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={219}
              name="t4X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={219}
            >
              4X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={236}
              name="t00"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={236}
            >
              00
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={237}
              name="t0"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={237}
            >
              0
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={238}
              name="t2"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={238}
            >
              2
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={239}
              name="t4"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={239}
            >
              4
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={240}
              name="t6"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={240}
            >
              6
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={241}
              name="t8"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={241}
            >
              8
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={242}
              name="t10"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={242}
            >
              10
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={243}
              name="t12"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={243}
            >
              12
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={244}
              name="t14"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={244}
            >
              14
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={245}
              name="t16"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={245}
            >
              16
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={246}
              name="t18"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={246}
            >
              18
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={247}
              name="t20"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={247}
            >
              20
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={248}
              name="t22"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={248}
            >
              22
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={249}
              name="t24"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={249}
            >
              24
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={250}
              name="t26"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={250}
            >
              26
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={251}
              name="t28"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={251}
            >
              28
            </label>
          </div>
        </div>
      </div>
      <div
        className="FormCheckboxSection"
      >
        <p>
          Maternity US sizing
        </p>
        <div
          className="FormCheckboxSection__options-container"
        >
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={316}
              name="mXXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={316}
            >
              XXS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={317}
              name="mXS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={317}
            >
              XS
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={318}
              name="mS"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={318}
            >
              S
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={319}
              name="mM"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={319}
            >
              M
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={320}
              name="mL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={320}
            >
              L
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={321}
              name="mXL"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={321}
            >
              XL
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={322}
              name="m2X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={322}
            >
              2X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={323}
              name="m3X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={323}
            >
              3X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={324}
              name="m4X"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={324}
            >
              4X
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={341}
              name="m00"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={341}
            >
              00
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={342}
              name="m0"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={342}
            >
              0
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={343}
              name="m2"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={343}
            >
              2
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={344}
              name="m4"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={344}
            >
              4
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={345}
              name="m6"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={345}
            >
              6
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={346}
              name="m8"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={346}
            >
              8
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={347}
              name="m10"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={347}
            >
              10
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={348}
              name="m12"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={348}
            >
              12
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={349}
              name="m14"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={349}
            >
              14
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={350}
              name="m16"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={350}
            >
              16
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={351}
              name="m18"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={351}
            >
              18
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={352}
              name="m20"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={352}
            >
              20
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={353}
              name="m22"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={353}
            >
              22
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={354}
              name="m24"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={354}
            >
              24
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={355}
              name="m26"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={355}
            >
              26
            </label>
          </div>
          <div
            className="FormCheckboxSection__option"
          >
            <input
              id={356}
              name="m28"
              onChange={[Function]}
              type="checkbox"
            />
            <label
              htmlFor={356}
            >
              28
            </label>
          </div>
        </div>
      </div>
    </form>
    <div />
  </section>
  <footer
    className="NPFFooter"
  >
    <button
      className="NPFFooter_previous"
      onClick={[Function]}
      type="button"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-arrow-left fa-w-14 NPFFooter_arrow"
        data-icon="arrow-left"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M257.5 445.1l-22.2 22.2c-9.4 9.4-24.6 9.4-33.9 0L7 273c-9.4-9.4-9.4-24.6 0-33.9L201.4 44.7c9.4-9.4 24.6-9.4 33.9 0l22.2 22.2c9.5 9.5 9.3 25-.4 34.3L136.6 216H424c13.3 0 24 10.7 24 24v32c0 13.3-10.7 24-24 24H136.6l120.5 114.8c9.8 9.3 10 24.8.4 34.3z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span>
        PREVIOUS
      </span>
    </button>
    <button
      className="NPFFooter_next"
      onClick={[Function]}
      type="button"
    >
      <span>
        NEXT
      </span>
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-arrow-right fa-w-14 NPFFooter_arrow"
        data-icon="arrow-right"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M190.5 66.9l22.2-22.2c9.4-9.4 24.6-9.4 33.9 0L441 239c9.4 9.4 9.4 24.6 0 33.9L246.6 467.3c-9.4 9.4-24.6 9.4-33.9 0l-22.2-22.2c-9.5-9.5-9.3-25 .4-34.3L311.4 296H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h287.4L190.9 101.2c-9.8-9.3-10-24.8-.4-34.3z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </button>
  </footer>
</div>
`;
